recipes-wpe/wpebackend-rdk/wpebackend-rdk_0.1.bb

```diff
@@ -33,7 +32,7 @@ PACKAGECONFIG ?= "${WPE_BACKEND} virtualinput"
 PACKAGECONFIG[imx6]             = "-DUSE_BACKEND_VIV_IMX6_EGL=ON,,imx-gpu-viv"
 PACKAGECONFIG[intelce]          = "-DUSE_BACKEND_INTEL_CE=ON,,intelce-display"
 PACKAGECONFIG[nexus]            = "-DUSE_BACKEND_BCM_NEXUS=ON,,broadcom-refsw"
-PACKAGECONFIG[rpi]              = "-DUSE_BACKEND_BCM_RPI=ON,,userland"
+PACKAGECONFIG[rpi]              = "-DUSE_BACKEND_BCM_RPI=ON,,mesa"
```

Comment:
It's better to have vc4graphics as new PACKAGECONFIG and select based on MACHINE_FEATURES.